Tests/MapboxCoreNavigationTests/RouteControllerTests.swift

```diff
@@ -65,3 +68,13 @@ extension RouteControllerTests: RouterDataSource {
         return NavigationLocationManager.self
     }
 }
+
+extension RouteControllerTests: RouterDelegate {
+    func router(_ router: Router, didRefresh routeProgress: RouteProgress) {
```

Comment:
Why did you add this implementation too?

Reply:
Xcode said that I must implement this method, I will remove and check again, Thanks.